packages/base-styles/_mixins.scss

```diff
@@ -335,6 +335,31 @@
 	}
 }
 
+// The editor input reset with increased specificity to avoid theme styles bleeding in.
+@mixin editor-input-reset() {
+	font-family: $editor-html-font !important;
+	color: $gray-900 !important;
+	background: $white !important;
+	padding: $grid-unit-15 !important;
+	border: $border-width solid $gray-900 !important;
+	box-shadow: none !important;
+	border-radius: $radius-block-ui !important;
+
+	/* Fonts smaller than 16px causes mobile safari to zoom. */
```

Comment:
If I recall correctly, `/* */` in the SCSS compiler is kept in place, whereas `//` is stripped out. I think we may have used this in the past for anyone reviewing the CSS files, not just the SCSS files. But in part, I think "View source" is increasingly not something people do, and in part I also think there's some minification that happens with mixins making all context lost here. So it might be good to change this to the `//` syntax.